Composer/packages/client/src/components/CreationFlow/v2/DefineConversation.tsx

```diff
@@ -154,7 +154,7 @@ const DefineConversationV2: React.FC<DefineConversationProps> = (props) => {
         if (!value || !nameRegexV2.test(`${value}`)) {
           // botName is used as used when generating runtime namespaces which cannot start with a number
           if (value && !isNaN(+value.toString().charAt(0))) {
-            return formatMessage('Bot name cannot not start with a number');
+            return formatMessage('Bot name cannot not start with a number or space');
```

Comment:
`cannot not` seems really confusing, is this intended or a typo?

Reply:
Ah oops, I have removed the extra `not`